### PR TITLE
main fixes

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
relates to pytest-dev/meta#8

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -199,7 +199,6 @@ Short version
 #. Follow **PEP-8** for naming and `black <https://github.com/psf/black>`_ for formatting.
 #. Tests are run using ``tox``::
 
-    tox -e linting,py37
```
Comment (nicoddemus) [inline CONTRIBUTING.rst:202]: Was this removed by accident? I think it should stay here.

**Review by nicoddemus — COMMENTED** [review]
Great work!

Just some comments:

* There's a reference to `master` in `development_guide.rst`.
* The diff shows an empty `HOWTORELEASE.rst` file?

**Comment by RonnyPfannschmidt** [comment]
thanks for the comments, my key mistake was starting off in the branch from the debugging session with @nedbat where i did not check my branch state

**Review by nicoddemus — APPROVED** [review]

**Comment by The-Compiler** [comment]
This seems very incomplete...

```
CHANGELOG.rst
7:The source document can be found at: https://github.com/pytest-dev/pytest/blob/master/doc/en/changelog.rst

README.rst
27:   :target: https://results.pre-commit.ci/latest/github/pytest-dev/pytest/master

RELEASING.rst
35:1. Create a new maintenance branch from ``master``::
38:        git branch 5.2.x upstream/master
51:1. Create a new maintenance branch from ``master``::
54:        git branch 6.0.x upstream/master
69:into ``master`` and ported back to the maintenance branch, even for release candidates.
86:   ``upstream/master`` and push it to ``upstream``.
117:#. Cherry-pick the CHANGELOG / announce files to the ``master`` branch::
120:       git checkout origin/master -b cherry-pick-release
125:#. For major and minor releases, tag the release cherry-pick merge commit in master with
128:       git checkout master

CONTRIBUTING.rst
237:    # now, create your own branch off "master":
390:#. First, make sure the bug is fixed the ``master`` branch, with a regular pull
392:   applicable to ``master`` anymore.
394:#. ``git checkout origin/1.2.x -b backport-XXXX`` # use the master PR number here
398:    nicoddemus merged commit 0f8b462 into pytest-dev:master
411:As mentioned above, bugs should first be fixed on ``master`` (except in rare occasions
420:   the backport, normally the one that merged the PR on ``master``.
421:4. If a non-maintainers notices a bug which is fixed on ``master`` but has not been backported

scripts/prepare-release-pr.py
6:``6.1.x`` or ``master``) and if it should be a major release.

doc/en/changelog.rst
6670:  master branch in git repo: "master" branch now keeps the bug fixes, changes
(debatable because it's historical)

scripts/release-on-comment.py
6:`master` and is triggered by two comment related events:
19:* a major release from master if "major" appears in the phrase in that position
20:* a feature or bug fix release from master (based if there are features in the current changelog
233:    if is_feature_release and base_branch != "master":
237:            created from `master`:
```

**Comment by RonnyPfannschmidt** [comment]
Thanks for the note, sounds like a good idea to add test coverage on those

**Pull request merged**